src/Propel/Generator/Builder/DataModelBuilder.php

```diff
@@ -262,11 +262,11 @@ public function getMultiExtendObjectBuilder(): MultiExtendObjectBuilder
      * @param \Propel\Generator\Model\Table $table
      * @param string $classname The class of builder
      *
-     * @return self
+     * @return static
```

Comment:
mb $this?

Reply:
$builder = new $classname($table);

It cannot possibly be the same object.